src/main/java/dk/cngroup/lentils/repository/HintTakenRepository.java

```diff
@@ -22,6 +22,14 @@
             nativeQuery = true)
     List<HintTaken> findAllByTeamAndCypher(Long teamId, Long cypherId);
 
+    @Query(value = "SELECT ht.* FROM hint_taken ht " +
```

Comment:
Pokud by to šlo, nepsal bych to jako native query. Ale přepsal bych to jen v JPQL, kde nemáš jména tabulek a sloupců, ale názvy entit a atributů v Javě. JPA pak vygeneruje korektní SQL dotaz s ohledem na použitou databázi.

Reply:
upraveno